src/components/ArticleInfo.jsx

```diff
@@ -1,17 +1,27 @@
 import { Link } from "react-router-dom"
-
+import { useEffect, useState} from "react"
+import axios from "axios"
 
 export default function ArticleInfo({article}) {
+    let [addVote, setAddVote] = useState(0)
 
```

Comment:
To improve code readability you could destructure some of article's properties here to avoid you having to say article. everything